### PR TITLE
[7.x.x Backport] Bugfix 1262887 - Transparent meshes are rendered as opaques when using Custom Render Pass

### PR DESCRIPTION
**Pull request by ellioman**
# Purpose of this PR
Backport of #1613

# Yamato
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fbackports%252Fbugfix-1262887-transparents-renderered-opaque

### CODE REVIEW

**Review by pbbastian — COMMENTED** [review]
Need status update on red CI before approving

**Comment by ellioman** [comment]
> Need status update on red CI before approving

I'm waiting for this PR to land in order to get green CI
https://github.com/Unity-Technologies/Graphics/pull/1910/

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [7.5.2] - 2020-09-14
 
 ### Fixed
+- Fixed not using the local skybox on the camera game object when Lighting>environment skybox was set to null.
+- Fixed an issue with a render texture failing assertion when chosing an invalid format. [case 1271034](https://issuetracker.unity3d.com/issues/the-error-occurs-when-a-render-texture-which-has-a-certain-color-format-is-applied-to-the-cameras-output-target)
```
Comment (pbbastian) [inline com.unity.render-pipelines.universal/CHANGELOG.md:11]: Seems like these two were moved unintentionally?

**Review by pbbastian — CHANGES_REQUESTED** [review]
Code looks good, looks like there was a merge issue in the change log

**Review by pbbastian — APPROVED** [review]

File: com.unity.render-pipelines.universal/CHANGELOG.md (hunk 1 above)
Comment (ellioman) [reply com.unity.render-pipelines.universal/CHANGELOG.md:11]: Sorry for not mentioning this in the PR description.

The PR's that added those lines came in after 7.5.1 was released. So they were incorrectly placed and will be in 7.5.2.
Decided to fix those in this PR as well.

**Comment by ellioman** [comment]
ABV was green before a changelog fix. I therefore cancelled the run.

**Pull request merged**